org/pr/installable-build.ts

```diff
@@ -131,7 +135,7 @@ async function getDownloadCommentText(status) {
   if (apkArtifacts.length == 1) {
     return `You can test the changes on this Pull Request by downloading the APK [here](${apkArtifacts[0].url}).`
   } else if (apkArtifacts.length > 1) {
-    const links = apkArtifacts.map(artifact => ` - [${artifact.path.split("/").pop()}](${artifact.url})`).join(`\n`)
+    const links = apkArtifacts.map(artifact => ` - [${artifact.path.split("/").pop()}](${artifact.url})\n<img src="${ generateQR( artifact.url )}" />`).join(`\n`)
```

Comment:
I'm late to the party (sorry I saw the review request but was too busy to get to it sooner), but what do you think (either in a next iteration… or more likely, when @jkmassel you'll re-implement Installable-Build Comments directly via buildkite pipeline) to use a collapsible section instead?

The odd thing with this current HTML is that it starts a bullet-point list but that list will contain large images (of the QR code) between the items. Not big a deal, but a bit unusual for bullet lists. But then there's also the fact that it can quickly take quite some vertical space in the comment, especially with 2 QR codes (one for WP, one for JP) one below the other.

So I'm suggesting something like this instead:

```suggestion
    const links = apkArtifacts.map(artifact => `<details><summary><a href="${artifact.url}">${artifact.path.split("/").pop()}</a></summary><img src="${generateQR(artifact.url)}" /></details>
```

Which should render something like below, where QR codes are hidden by default to reduce vertical space, but can be shown inline with a single click. WDYT?

---

You can test the changes on this Pull Request by downloading the APKs:
<details><summary><a href="https://117588-9306568-gh.circle-artifacts.com/0/Artifacts/Jetpack-pr-15683-build-117588.apk">Jetpack-pr-15683-build-117588.apk</a></summary><img src="https://chart.googleapis.com/chart?chs=150x150&cht=qr&chl=https%3A%2F%2F117588-9306568-gh.circle-artifacts.com%2F0%2FArtifacts%2FJetpack-pr-15683-build-117588.apk&choe=UTF-8" /></details>
<details><summary><a href="https://117588-9306568-gh.circle-artifacts.com/0/Artifacts/WordPress-pr-15683-build-117588.apk">WordPress-pr-15683-build-117588.apk</a></summary><img src="https://chart.googleapis.com/chart?chs=150x150&cht=qr&chl=https%3A%2F%2F117588-9306568-gh.circle-artifacts.com%2F0%2FArtifacts%2FWordPress-pr-15683-build-117588.apk&choe=UTF-8" /></details>